end_to_end/tpu/mixtral/8x7b/1_test_mixtral.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# This file, combined with step 2 in the same directory, demonstrates:
+# 1. Converts the Mistral PyTorch checkpoint to MaxText(orbax) format using a CPU VM.
+# 2. Takes the MaxText (orbax) checkpoint to run inference and fine-tuning on a TPU VM.
+
+# The flow of this file is to convert the Mistral PyTorch checkpoint to MaxText (orbax) format using a CPU VM.
+
+# Example Usage: export BASE_OUTPUT_PATH=/path/to/GCS/bucket; bash end_to_end/tpu/mixtral/8x7b/1_test_mixtral.sh
+# Use the same BASE_OUTPUT_PATH for both 1_test_mixtral.sh & 2_test_mixtral.sh.
+
+set -ex
+MODEL_VARIATION='8x7b'
+
+if [ -z "${BASE_OUTPUT_PATH}" ]; then
+    # Non-Googlers please remember to point BASE_OUTPUT_PATH to GCS buckets that you own, this script uses internal buckets for testing.
+    export BASE_OUTPUT_PATH=gs://runner-maxtext-logs/$(date +%Y-%m-%d-%H-%M)/
+    echo "BASE_OUTPUT_PATH is not set, using BASE_OUTPUT_PATH = ${BASE_OUTPUT_PATH}"
+fi
+
+# Download checkpoint, convert it to MaxText(orbax) format
+pip3 install torch
+gcloud storage cp -r gs://maxtext-external/mixtral-8x7B-v0.1-Instruct /tmp
+JAX_PLATFORMS=cpu python3 MaxText/llama_or_mistral_ckpt.py --base-model-path /tmp/mixtral-8x7B-v0.1-Instruct --model-size mixtral-8x7b --maxtext-model-path ${BASE_OUTPUT_PATH}${MODEL_VARIATION}/decode-ckpt-maxtext/
```

Comment:
Do we not want to have an unscanned version of the checkpoint for efficient decoding?

https://github.com/google/maxtext/blob/db31dd4b0b686bca4cd7cf940917ec372faa183a/end_to_end/tpu/gemma/7b/1_test_gemma.sh#L42

Reply:
I think this ckpt is used for both decoding and fine-tuning in the 2nd script. Do you suggest we add `generate_param_only_checkpoint.py` test for decoding purpose? If so, can I add more tests in a coming PR? I think the main goal of this PR to split the `test_mixtral.sh` into 2, and we could definitely add more tests. Please feel free to let me know your thoughts!

Reply:
yes `generate_param_only_checkpoint.py` helps getting unscanned checkpoint for faster decoding, whereas the `llama_or_mistral_ckpt.py` generates a scanned checkpoint where decoding definitely works albeit a bit slower.

Yes, please go ahead and add the other functionalities like the gemma-7b/llama2-7b tests in coming PRs. Thanks Ran!